solver.rb

```diff
@@ -0,0 +1,31 @@
+class Solver
+  def factorial(number)
+    if number.negative?
+      'Plese give me a positive number'
```

Comment:
- [ ] As demanded by the project requirements, kindly make this method raise an exception error if the number is negative. Also, test for the error raised.

you can use;
```ruby
if number.negative?
  raise Exception
```